src/TakeoffController.cpp

```diff
@@ -108,14 +108,14 @@ bool TakeoffController::update(const ros::Time& time,
         ROS_ERROR("Tried to update TakeoffHandler with time before last update");
         return false;
     }
-
+    bool success = true;
     // Get the current transform (xyz) of the quad
-    geometry_msgs::TransformStamped transform;
-    bool success = transform_wrapper_.getTransformAtTime(transform,
-                                                         "map",
-                                                         "base_footprint",
-                                                         time,
-                                                         update_timeout_);
+    //geometry_msgs::TransformStamped transform;
```

Comment:
why is this all commented out?

Reply:
It needed to be completely removed actually which I just did.